### PR TITLE
Upgrade: All deps

### PR DESCRIPTION
**Pull request by omgaz**
- [x] Need to check with adslot products
- [x] Remove temp alexandria sha (and release alexandria)
- [x] IE Testing
- [x] Remove dist files

### CODE REVIEW

File: scripts/git-hooks/pre-commit [hunk 1]
```diff
@@ -8,12 +8,17 @@ then
   isVersionChange=$(git diff --cached | grep "+  \"version\"")
   if [[ -n "${isVersionChange[@]}" ]]
   then
+    echo -e "---------------------\nPreparing for release\n---------------------"
+    echo "👾  Installing node modules."
+    npm install
+    echo "📑  Compiling distribution files for release."
```
Comment (OndrejCholeva) [inline scripts/git-hooks/pre-commit]: This is how compiler looks like :-)

File: src/components/adslotUi/SplitPaneComponent.jsx [hunk 2]
```diff
@@ -7,9 +7,11 @@ require('styles/adslotUi/SplitPane.scss');
 
 const SplitPaneComponent = ({ children, dts, additionalClassNames }) => {
   const splitPaneClass = classNames('splitpane-component', ...additionalClassNames);
+  const dtsProp = {};
+  if (dts) dtsProp['data-test-selector'] = dts;
 
   return (
-    <div className={splitPaneClass} if dts data-test-selector={dts}>
+    <div className={splitPaneClass} {...dtsProp}>
```
Comment (OndrejCholeva) [inline src/components/adslotUi/SplitPaneComponent.jsx]: can't we use the  same `expandDts` helper as in alexandria?

File: scripts/git-hooks/pre-commit (hunk 1 above)
Comment (omgaz) [inline scripts/git-hooks/pre-commit]: You approved this in Alexandria, quit being so indecisive ;)

**Comment by patrick-ivers** [comment]
Merged! 🌋

**Pull request merged**